client/js/app/main/modules/menu/index.js

```diff
@@ -1,7 +1,8 @@
 var angular = require('angular');
 
 var menuModule = angular.module('menuModule', [
-    require('./projectList')
+    require('./projectList'),
+    require('./createProject')
 ])
```

Comment:
createProject module should be in main/modules folder and be exported from main/modules/index.js